Forms Automated Test - LPS-133214 | master

fmarcio opened the pull request.

CalumR23 commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-17.liferay.com/job/test-portal-source-format/5241/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-source-format/builds/5241/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ba9527cfa360ea0afe56ce720d28db8943fb4954">ba9527cfa360ea0afe56ce720d28db8943fb4954</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/fmarcio/liferay-portal/tree/LPS-133214">LPS-133214</a><br/>Branch GIT ID: <a href="https://github.com/fmarcio/liferay-portal/commit/9983cbcd3f62cf4be7c19d72ca8c874e07c46a7a">9983cbcd3f62cf4be7c19d72ca8c874e07c46a7a</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-17.liferay.com/job/test-portal-source-format/5241/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-17.liferay.com/userContent/jobs/test-portal-source-format/builds/5241/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-17.liferay.com/job/test-portal-source-format/5241/">test-portal-source-format#5241</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-17.liferay.com/userContent/jobs/test-portal-source-format/builds/5241/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-forms/liferay-portal/pull/496">liferay-forms#496</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1538970984">[master] ci:test:sf - fmarcio &gt; liferay-forms - PR#496 - 2021-06-09[15:42:39]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-17.liferay.com/job/publish-testray-report/1055/">publish-testray-report#1055</a><br/></div></div>

fmarcio commented:
@CalumR23 I added the pending-sender label because this PR uses `FormViewBuilder.macro ` and it seems that it does not belong to forms. I'll fix it soon and then update this PR. Thanks :-)

fmarcio commented:
> @CalumR23 I added the pending-sender label because this PR uses `FormViewBuilder.macro ` and it seems that it does not belong to forms. I'll fix it soon and then update this PR. Thanks :-)

Update June 10: I removed the "pending-sender" label since it will not necessary update this PR anymore (`FormViewBuilder.macro` will be avoided only on future PRs)

CalumR23 commented:
[Combined PR](https://github.com/CalumR23/liferay-portal/pull/5)
